GOVERNANCE.md

```diff
@@ -234,6 +234,8 @@ A TSC member guides the direction of the project and ensures a healthy future fo
 
 A TSC member has significant sway in software design decisions. For this reason, coding experience is critical for this role. TSC membership is one of the only roles that requires a significant contribution history of code to the Astro project on GitHub.
 
+TSC seats are not time-limited. The size of the TSC can not be larger than 5 members. This is to ensure that the TSC is able to make decisions efficiently.
```

Comment:
@sarah11918 It's stated by the governance that TSC members are self-nominated and approved by the Steward. 

This means that even after (and if) this PR lands, the criteria for who steers the ship won't change. Regarding your concern, I think it should be discussed as part of a different issue/PR.